products.py

```diff
@@ -1,62 +1,177 @@
+            'Binoculars','Campfire Grill','Multi-tool','Camping Chairs','Portable Water Filter','Map and Compass','Flashlight','Fire Starter','Sunscreen','Bug Spray','Water Bottle',
+            'Portable Charger']
+
+pet_list = ['Pet Food','Water Bowl','Food Bowl','Pet Bed','Leash','Collar','Harness','Grooming Brush','Cat Litter','Litter Box','Pet Carrier','Toys','Scratching Post','Pet Shampoo',
+            'Dental Chews','Pet Brush','Pet Nail Clippers','Pet Crate','Feeding Station','Pet Gates','Pet Cleaning Wipes','Training Pads','Pet Backpack','Pet Clothes','Pet Carrier Bag']
+
+software_list = ['Windows 11','macOS Ventura','Ubuntu','Fedora','Android','iOS', 'Adobe Photoshop']
+
+sports_list = ['Soccer Ball','Basketball','Baseball Glove','Baseball Bat','Volleyball','Tennis Racket','Golf Clubs','Golf Balls','Golf Tees','Golf Glove',
+            'Golf Bag','Rangefinder']
+
+toys_list = ['Soccer Ball','Basketball','Baseball Glove','Baseball Bat','Volleyball','Tennis Racket','Golf Clubs','Golf Balls','Golf Tees','Golf Glove',
+            'Golf Bag','Rangefinder']
+
+video_game_list = ['The Legend of Zelda: Breath of the Wild','Grand Theft Auto V','Red Dead Redemption 2','Assassin\'s Creed Valhalla','Spider-Man (PS4)','Horizon Forbidden West',
+                        'Call of Duty: Modern Warfare II','Battlefield 2042','DOOM Eternal','Overwatch 2','Counter-Strike: Global Offensive','Apex Legends']
+
+video_list = ['Mad Max: Fury Road','Inception','The Dark Knight','Avengers: Endgame','Jurassic Park','Mission: Impossible – Fallout','Star Wars: Episode IV - A New Hope',
+                        'Blade Runner 2049','The Matrix','Avatar','The Lord of the Rings: The Fellowship of the Ring','Guardians of the Galaxy']
+
+watch_list = ['Rolex Submariner','Omega Seamaster','Patek Philippe Nautilus','Audemars Piguet Royal Oak','Tag Heuer Carrera','Breitling Navitimer','Longines Master Collection',
+                    'Omega Speedmaster Professional','IWC Pilot’s Watch','Jaeger-LeCoultre Reverso','Cartier Tank','Tissot Le Locle']
+
+def generateProductId():
     num  = range(1,2000)
     productID = random.sample(num,1)[0]
     return productID
 
-def generate_name():
-    return fake.name()
+def getProductCategoryProvider():
+    product_category_provider = DynamicProvider(
+        provider_name = 'category',
+        elements = ['Beauty', 'Books', 'Electronics', 'Fine Art', 'Health & Personal Care', 
+                        'Home & Garden', 'Music & DVD', 'Musical Instruments', 'Office Products', 
+                        'Outdoors', 'Pet Supplies', 'Software', 'Sports', 'Toys & Games', 'Video Games', 
+                        'Video & Blu-ray', 'Watches'],
+    )
+    return product_category_provider
 
 # 17 categories
 def grab_random_category_product(fake):
     selection = random.randint(1,17)
+    #print('Selection: ' + str(selection))
```

Comment:
Please remove commented out code. There's multiple instances in products.py so just make sure you get them all

Reply:
We can use the pyint function and move forward